c10/core/TensorImpl.cpp

```diff
@@ -48,10 +48,14 @@ TensorImpl::TensorImpl(
     Storage&& storage,
     DispatchKeySet key_set,
     const caffe2::TypeMeta data_type)
-    : TensorImpl(std::move(storage), key_set, data_type, storage.device()) {}
+    : TensorImpl(std::move(storage), key_set, data_type, storage.device()) {
+  init_bitfields();
```

Comment:
These seem unnecessary, if we call the TensorImpl constructor that should take care of initialization, no?

Reply:
Oops yes, thanks for catching